src/utils/wallet.js

```diff
@@ -151,11 +151,13 @@ export class Wallet {
          '', // fundingOwner
          0 // fundingAmount
       )
-      const parsedUrl = new URL(successUrl)
-      parsedUrl.searchParams.set('account_id', accountId)
-      parsedUrl.searchParams.set('public_key', publicKey)
-      const redirectUrl = parsedUrl.href
-      window.location.href = redirectUrl
+      if (successUrl) {
```

Comment:
to provide good UX there still needs to be success URL, but pointing to localhost, so flow is like:

1. Run `near login`
2. `near-shell` starts webserver on `localhost` listening for callback and prints login URL in console. `successUrl` should point to this webserver.
3. Click login URL in console.
4. Chose account in wallet and get redirected to `localhost`.
5. `near-shell` handles callback and exits.

Reply:
yeah that's why PR description says it's V0 and it's a todo

Reply:
I really don't think we want to be starting webservers for this. People may have weird firewall setups/shells we are not expecting. Having a complex system like this will just make it not work period for those use cases. 
We can ask them to rerun login command with a diff account id if the orginal account id is not available. In any case this is all not V0.